### PR TITLE
version-extractor-android 0.0.3

### PR DESCRIPTION
**Pull request by vladimirpetrovski**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by lpusok** [comment]
@vladimirpetrovski Thanks for the contribution. Please merge this change to allow the CI to pass: https://github.com/vladimirpetrovski/bitrise-steplib/pull/1.

File: steps/version-extractor-android/0.0.3/step.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+is_skippable: false
+run_if: ""
+inputs:
+- build_gradle_path: $BITRISE_SOURCE_DIR/app/build.gradle
+  opts:
+    is_required: true
+    summary: Path to the app's build.gradle file where the versionCode and versionName
+      are defined.
+    title: Path to the build.gradle file
+- gradlew_path: $GRADLEW_PATH
```
Comment (lpusok) [inline steps/version-extractor-android/0.0.3/step.yml:38]: Would recommend setting this to `./gradlew` or an empty string as $GRADLEW_PATH is not set by default. I found it used in the default gradle-runner step, but I believe it to be there by mistake.

Comment (vladimirpetrovski) [reply steps/version-extractor-android/0.0.3/step.yml:38]: Sure, I'll change the default value to `./gradlew` 👍

**Comment by vladimirpetrovski** [comment]
@lpusok I've created a new pull request on the latest changes 👉  #2485 . I'm closing this one.

**Pull request closed without merging**